src/main/java/tds/exam/results/services/impl/ExamResultsServiceImpl.java

```diff
@@ -50,6 +57,22 @@ public ExamResultsServiceImpl(final ExamService examService,
     @Override
     public TDSReport findAndSendExamResults(final UUID examId) {
         TDSReport report = new TDSReport();
+
+        if (properties.isRetryOnError()) {
+            findAndSendExamResults(examId, report);
+        } else {
+            try {
```

Comment:
I asked Ernie to add this so we don't have to drain the queues when we're doing testing.  At some point we'll remove it.